x-pack/plugin/esql/qa/testFixtures/src/main/resources/meta.csv-spec

```diff
@@ -31,6 +31,7 @@ greatest                 |"integer|long|double|boolean|keyword|text|ip|version g
 least                    |"integer|long|double|boolean|keyword|text|ip|version least(first:integer|long|double|boolean|keyword|text|ip|version, ?rest...:integer|long|double|boolean|keyword|text|ip|version)"        | first           |"integer|long|double|boolean|keyword|text|ip|version"            |""                                            |"integer|long|double|boolean|keyword|text|ip|version"                    | "Returns the minimum value from many columns."                      | false       | true | false
 left                     |"keyword left(string:keyword|text, length:integer)"                |[string, length]         |["keyword|text", "integer"]            |["The string from which to return a substring.", "The number of characters to return."]                                            |keyword                    | "Returns the substring that extracts 'length' chars from 'string' starting from the left."                      | [false, false]       | false | false
 length                   |"integer length(string:keyword|text)"                                        |string                     |"keyword|text"                 | ""                                                 |integer                    | "Returns the character length of a string."                      | false                | false | false
+locate                   |"integer locate(str:keyword|text, substr:keyword|text)"        |[str,substr]       |["keyword|text", "keyword|text"]            |["", ""]                                            |integer                    | "Returns an integer that indicates the position of a keyword substring within another string"                      | [false, false]       | false | false
```

Comment:
I'm sorry for this test. It was a good idea, but I think it's grown to be a problem. @luigidellaquila, would you be ok if I broke this test apart into a few tests so updating it is easier? Right now it's a huge blob of text to work through.

Reply:
+100, updating this test is always painful for me as well

Reply:
👍  it's a good idea to break it down into smaller tests, imho.

Reply:
I'm doing it. Now. Well. Maybe in an hour.